dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventOperationParamsMapper.java

```diff
@@ -196,6 +204,27 @@ public EventSearchParams map(EventOperationParams operationParams)
         .setIncludeDeleted(operationParams.isIncludeDeleted());
   }
 
+  private List<OrganisationUnit> validateAccessibleOrgUnits(
```

Comment:
nit: I would put this after `validateRequestedOrgUnit` so its in call order. I would also put `user` first as we are validating the user has access to the orgUnit, ... We are doing that in the ACL related services with `hasAccess(User user, ...)`/canAccess(User user, ...)` 😄